### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by johnkerl**
Typo on #115

### CODE REVIEW

File: R/manual_layer_udf.R [hunk 1]
```diff
@@ -47,6 +47,7 @@
 execute_generic_udf <- function(udf=NULL, registered_udf_name=NULL, args=NULL, result_format='native',
   args_format='native', namespace=NULL, language='r', resource_class=NULL)
 {
+  cat("EXECUTE_GENERIC_UDF RESOURCE_CLASS", resource_class, "\n")
```
Comment (eddelbuettel) [inline R/manual_layer_udf.R:50]: Maybe remove this / make it conditional on a debug flag?

**Review by eddelbuettel — APPROVED** [review]
Looks like a winner modulo one verbose message ripe for sheltering / removal?

File: R/manual_layer_udf.R (hunk 1 above)
Comment (johnkerl) [reply R/manual_layer_udf.R]: sorry this was debug stuff -- removed on a subsequent commit

**Comment by johnkerl** [comment]
Screenshot 2023-03-23 at 1 26 54 PM

**Pull request merged**